fvtest/compilertriltest/LogicalTest.cpp

```diff
@@ -103,3 +115,51 @@ INSTANTIATE_TEST_CASE_P(LogicalTest, Int32LogicalBinary, ::testing::Combine(
         std::make_tuple("iand", iand),
         std::make_tuple("ixor", ixor)
         )));
+
+class Int64LogicalBinary : public TRTest::BinaryOpTest<int64_t> {};
+
+TEST_P(Int64LogicalBinary, UsingConst) {
+    auto param = TRTest::to_struct(GetParam());
+
+    char inputTrees[120] = {0};
+    std::snprintf(inputTrees, 120, "(method return=Int64 (block (lreturn (%s (lconst %li) (lconst %li)) )))", param.opcode.c_str(), param.lhs, param.rhs);
```

Comment:
In new tests, we've been trying to encourage splitting Tril code into multiple lines to make it easier to ready. Would you mind breaking up this string into multiple lines?